ketcher-autotests/tests/Reaction-tool/Mapping-Unmapping tool/mapping-tool.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { test } from '@playwright/test';
+import {
+  delay,
+  takeEditorScreenshot,
+  LeftPanelButton,
+  clickInTheMiddleOfTheScreen,
+  selectLeftPanelButton,
+  selectNestedTool,
+  ReactionMappingTool,
+  DELAY_IN_SECONDS,
+  openFileAndAddToCanvas,
+  selectRingButton,
+  RingButton,
+  getCoordinatesTopAtomOfBenzeneRing,
+  selectTopPanelButton,
+  TopPanelButton,
+  dragMouseTo,
+} from '@utils';
+import { getAtomByIndex } from '@utils/canvas/atoms';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Click atoms to map atoms in a reaction', async ({ page }) => {
+    /* Test case: EPMLSOPKET-1799, EPMLSOPKET-8909
+    Description:  Click atoms to map atoms in a reaction
+    */
+    await openFileAndAddToCanvas('reaction_3.rxn', page);
```

Comment:
There is still my proposal: can we name `reaction_3.rxn` in more meaningful way?
Can't we reuse existing files? If this file contains some special data, then which?